apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-card.js

```diff
@@ -86,9 +87,16 @@ function WelcomeTourCard( {
 }
 
 function CardNavigation( { cardIndex, lastCardIndex, onDismiss, setCurrentCardIndex } ) {
+	const localeSlug = useLocale();
+
 	// These are defined on their own lines because of a minification issue.
 	// __('translations') do not always work correctly when used inside of ternary statements.
-	const startTourLabel = __( 'Start Tour', 'full-site-editing' );
+	const startTourLabel =
+		localeSlug === 'en' ||
+		localeSlug === 'en-gb' ||
+		hasTranslation( 'Try it out!', 'full-site-editing' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **6** times:
`translate( 'Try it out: ' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/etk-welcome-guide-1st-step)